### ISSUE TITLE
Accessible Authentication (3.3.7) confusing example (properly marked up email and password)

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#1363]
While the understanding text https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication.html says that the fundamental problem this SC tries to solve is login processes that rely on the user remembering a *username* and *password*, it then provides as the first (positive, presumably) example

> A web site uses a properly marked up email and password as the login authentication.

Why would remembering the *email* and password be ok while having to remember *username* and password makes a site fail at level A? Is the underlying assumption that *email* is easier to remember than *username*?

Moreover, what does "properly marked up" even mean?

Either this example lacks a lot more context/explanation, or it's built on the assumption that people will have an easier time remembering their email rather than a username? And for both, "password" seems to be ok then?

As relying on username/password is essentially what the vast majority of sites rely on today for authentication, this particular scenario should really be explained in much more depth. Explaning in particular what it is that all those sites that currently rely on username/password should do, very specifically, to not fail at level A here. Is it just a case of "make sure there's also a username/password recovery at the very least"?

**Comment by alastc** [comment]
> Why would remembering the email and password be ok while having to remember username and password makes a site fail at level A? Is the underlying assumption that email is easier to remember than username?

That isn't the intent, a simple fix would be to use username/password in the example, e.g:
> A web site uses a properly marked up username and password as the login authentication. Copy &amp; paste is not blocked on the inputs, so the user's browser or extension can identify the purpose of the inputs and automatically fill in the username and password.

Also, did you see the note in the intent?
> Web sites can employ username (or email) and password inputs as an authentication method if it meets <a href="#">Success Criterion 1.3.5 Input Purpose</a> and password managers / browser functionality is not blocked. When properly marked up, browser features or password managers can save the user’s information and refill the login.

**Comment by alastc** [comment]
I created a branch (#1415) for updates based on this, starting with the above comment.
